devnet-sdk/book/.gitignore

```diff
@@ -0,0 +1 @@
+book
```

Comment:
Was adding this .gitignore intentional?

Reply:
Nevermind, I see that this is ignoring any subdirs `devnet-sdk/book/book/`. My editor has a bug and is taking this to mean that `devnet-sdk/book/` should be ignored.